src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,49 @@
 import React from 'react';
-
+import { Movie } from '../../types/movies';
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+interface Props {
+  movie: Movie;
+}
+
+export const MovieCard: React.FC<Props> = ({ movie }) => {
+  const {
+    title,
+    description,
+    imdbUrl,
+    imgUrl,
+  } = movie;
```

Comment:
why not de-structure right in params?
```suggestion
export const MovieCard: React.FC<Props> = ({
    title,
    description,
    imdbUrl,
    imgUrl,
  }) => {
```

Reply:
I find it less confusing if I destructure them separately because I can keep track of the input names between different layers.